frontend/views/components/tabs/TabWrapper.vue

```diff
@@ -25,6 +21,11 @@
           .c-icons
             i.icon-chevron-right
 
+        hr.tab-nav-separator
+
+      .tab-nav-list.is-subtitle Version 1.6.0
```

Comment:
1. If you're going to add a user-facing version it needs to match the actual version
2. Missing `i18n`